vite.config.js

```
@@ -7,11 +7,26 @@ import { fileURLToPath } from "url";
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
+// ★ MOCK 모드 전환 방법:
+//   - ON  (더미 데이터 사용) : 아래 mockAliases 객체 정의 줄과 스프레드 줄을 주석 해제
+//   - OFF (실제 API 사용)   : 해당 두 줄을 주석 처리
+const mockAliases = {
```

ESLint

[error] 'mockAliases' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)